{"uid":"76720fc1022a849b","name":"test_network_cut","fullName":"tests.integration.ha_tests.test_self_healing#test_network_cut","historyId":"6b9f6734f6f55bf616f6dd52eb37e3bd","time":{"start":1719887915241,"stop":1719888219876,"duration":304635},"description":"Completely cut and restore network.","descriptionHtml":"<p>Completely cut and restore network.</p>\n","status":"failed","statusMessage":"AssertionError: Unable to create continuous_writes table","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/tests/integration/ha_tests/test_self_healing.py\", line 390, in test_network_cut\n    await start_continuous_writes(ops_test, app)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/tests/integration/ha_tests/helpers.py\", line 778, in start_continuous_writes\n    for attempt in Retrying(stop=stop_after_delay(60 * 5), wait=wait_fixed(3), reraise=True):\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 437, in __iter__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 370, in iter\n    result = action(retry_state)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 412, in exc_check\n    raise retry_exc.reraise()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 185, in reraise\n    raise self.last_attempt.result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 451, in result\n    return self.__get_result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 403, in __get_result\n    raise self._exception\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/tests/integration/ha_tests/helpers.py\", line 786, in start_continuous_writes\n    assert action.results[\"result\"] == \"True\", \"Unable to create continuous_writes table\"\nAssertionError: Unable to create continuous_writes table","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"ops_test","time":{"start":1719883793261,"stop":1719883793508,"duration":247},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1719883793261,"stop":1719883793261,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1719883793261,"stop":1719883793261,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"primary_start_timeout","time":{"start":1719885059059,"stop":1719885063230,"duration":4171},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1719887915239,"stop":1719887915240,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1719887915240,"stop":1719887915240,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"continuous_writes::finalizer","time":{"start":1719888219893,"stop":1719888522066,"duration":302173},"status":"failed","statusMessage":"AssertionError: Unable to clear up continuous_writes table\nassert 'False' == 'True'\n  \n  - True\n  + False\n","statusTrace":"  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 300, in finalizer\n    event_loop.run_until_complete(async_finalizer())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 292, in async_finalizer\n    await gen_obj.__anext__()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/tests/integration/ha_tests/conftest.py\", line 29, in continuous_writes\n    for attempt in Retrying(stop=stop_after_delay(60 * 5), wait=wait_fixed(3), reraise=True):\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 437, in __iter__\n    do = self.iter(retry_state=retry_state)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 370, in iter\n    result = action(retry_state)\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 412, in exc_check\n    raise retry_exc.reraise()\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 185, in reraise\n    raise self.last_attempt.result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 451, in result\n    return self.__get_result()\n  File \"/usr/lib/python3.10/concurrent/futures/_base.py\", line 403, in __get_result\n    raise self._exception\n  File \"/home/ubuntu/github-runner/_work/test-runners-2-is-x64-postgresql-operator/test-runners-2-is-x64-postgresql-operator/tests/integration/ha_tests/conftest.py\", line 37, in continuous_writes\n    assert action.results[\"result\"] == \"True\", \"Unable to clear up continuous_writes table\"\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1719888522069,"stop":1719888522069,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"primary_start_timeout::finalizer","time":{"start":1719889127940,"stop":1719889128105,"duration":165},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1719889128110,"stop":1719889129357,"duration":1247},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1719889129360,"stop":1719889129362,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1719889129363,"stop":1719889129363,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1719889129363,"stop":1719889129364,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_self_healing"},{"name":"host","value":"large-542-ee541355f942afaeba810cbc"},{"name":"thread","value":"11194-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_self_healing"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"a2145eec7de1b9f1","status":"unknown","time":{}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":2,"broken":4,"skipped":0,"passed":17,"unknown":2,"total":25},"items":[{"uid":"17a3974227bd26a0","reportUrl":"../95//#testresult/17a3974227bd26a0","status":"passed","time":{"start":1719799189293,"stop":1719799639562,"duration":450269}},{"uid":"5d5809d62677d6fc","reportUrl":"../94//#testresult/5d5809d62677d6fc","status":"passed","time":{"start":1719713475607,"stop":1719714055457,"duration":579850}},{"uid":"cd2341790a4cd1ee","reportUrl":"../93//#testresult/cd2341790a4cd1ee","status":"passed","time":{"start":1719626588358,"stop":1719627045514,"duration":457156}},{"uid":"238e78dfe4c96952","reportUrl":"../92//#testresult/238e78dfe4c96952","status":"passed","time":{"start":1719540429726,"stop":1719541015027,"duration":585301}},{"uid":"4ae4c401a34cad3e","reportUrl":"../91//#testresult/4ae4c401a34cad3e","status":"passed","time":{"start":1719454144226,"stop":1719454690110,"duration":545884}},{"uid":"210ea60bd5e45d3c","reportUrl":"../88//#testresult/210ea60bd5e45d3c","status":"passed","time":{"start":1719367425269,"stop":1719367844401,"duration":419132}},{"uid":"df9557eb6bd87e90","reportUrl":"../87//#testresult/df9557eb6bd87e90","status":"passed","time":{"start":1719281213633,"stop":1719281884554,"duration":670921}},{"uid":"29e206b5b3f97e01","reportUrl":"../86//#testresult/29e206b5b3f97e01","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/2","time":{"start":1719194966964,"stop":1719195194593,"duration":227629}},{"uid":"35ee8fdd40673c46","reportUrl":"../85//#testresult/35ee8fdd40673c46","status":"passed","time":{"start":1719108366033,"stop":1719109011930,"duration":645897}},{"uid":"3ffd7f6ae06a9b0f","reportUrl":"../84//#testresult/3ffd7f6ae06a9b0f","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x735ba13c2620 state=finished raised AssertionError>]","time":{"start":1719026182553,"stop":1719026364647,"duration":182094}},{"uid":"97fc8b8048789e7d","reportUrl":"../83//#testresult/97fc8b8048789e7d","status":"unknown","time":{}},{"uid":"acefa1425bc910b7","reportUrl":"../82//#testresult/acefa1425bc910b7","status":"failed","statusDetails":"AssertionError: Unable to create continuous_writes table","time":{"start":1718852477572,"stop":1718852779181,"duration":301609}},{"uid":"c92da13bd748a05b","reportUrl":"../81//#testresult/c92da13bd748a05b","status":"passed","time":{"start":1718762672559,"stop":1718763079140,"duration":406581}},{"uid":"f9cc02a18a716eda","reportUrl":"../80//#testresult/f9cc02a18a716eda","status":"passed","time":{"start":1718678706115,"stop":1718679382784,"duration":676669}},{"uid":"a627e69c91d6204a","reportUrl":"../79//#testresult/a627e69c91d6204a","status":"passed","time":{"start":1718590154295,"stop":1718590607809,"duration":453514}},{"uid":"f183c237af565343","reportUrl":"../78//#testresult/f183c237af565343","status":"passed","time":{"start":1718503602936,"stop":1718504007027,"duration":404091}},{"uid":"a948a43ccdfb4e5","reportUrl":"../77//#testresult/a948a43ccdfb4e5","status":"passed","time":{"start":1718419889365,"stop":1718420570562,"duration":681197}},{"uid":"6f20f0fe7d6ef4ab","reportUrl":"../76//#testresult/6f20f0fe7d6ef4ab","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7e890cee7e80 state=finished raised AssertionError>]","time":{"start":1718331741743,"stop":1718331936230,"duration":194487}},{"uid":"722c0ce1b1f3df09","reportUrl":"../75//#testresult/722c0ce1b1f3df09","status":"passed","time":{"start":1718246025897,"stop":1718246676054,"duration":650157}},{"uid":"f0eae4a956c9de67","reportUrl":"../74//#testresult/f0eae4a956c9de67","status":"passed","time":{"start":1718159007096,"stop":1718159417173,"duration":410077}}]},"tags":["asyncio","group(1)"]},"source":"76720fc1022a849b.json","parameterValues":[]}